functions/file_processing/main.py

```diff
@@ -70,17 +69,27 @@ def preprocessing(bucket_name, blob_name):
         file=bytesIO
     )
 
+
 def df_from_store(bucket_name, blob_name):
     path = 'gs://{}/{}'.format(bucket_name, blob_name)
     if blob_name.endswith('.xlsx'):
-        df = pd.read_excel(path, dtype = str)
+        df = pd.read_excel(path, dtype=str)
     elif blob_name.endswith('.json'):
-        df = pd.read_json(path, dtype=False)
+        try:
+            df = pd.read_json(path, dtype=False)
+        except Exception as e:
+            logging.warning('Could not load valid json, trying to normalize')
+            bucket = client.get_bucket(bucket_name)
+            blob = storage.Blob(blob_name, bucket)
+            content = blob.download_as_string()
+            data = json.loads(content.decode('utf-8'))
+            df = pd.DataFrame.from_records(data['Rows'])
```

Comment:
Ik zou 'rows' niet hardcoden in de eventsourcing helpers maar in de config vermelden...